sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/AppendBlobAPITest.groovy

```diff
@@ -57,6 +57,9 @@ class AppendBlobAPITest extends APISpec {
         bu.create(headers, null, null, null)
         Response<BlobProperties> response = bu.getProperties()
 
+        // If the value isn't set the service will automatically set it
+        contentType = (contentType == null) ? "application/octet-stream" : contentType
```

Comment:
What's the reason you remove it from APISpec but added here?

Reply:
This isn't true for all cases that used the method